src/entities/mellow/mellowLpRouter.ts

```diff
@@ -840,6 +858,18 @@ class MellowLpRouter {
   public get autoRolloverRegistrationGasFeeUSD() {
     return this.autoRolloverRegistrationGasUnits * this.gasUnitPriceUSD;
   }
+
+  public canManageVaultPosition = (vaultIndex: number): boolean => {
+    if (this.canManageVaultPositions === undefined) {
+      return false;
+    }
+
+    if (!(vaultIndex < this.vaultsCount)) {
```

Comment:
why not write a more readable code?

```suggestion
    if (vaultIndex >= this.vaultsCount) {
```